src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -209,10 +209,26 @@ class BuiltinLower : public StmtExprMutator {
       return MakeArray(op);
     } else if (op->op.same_as(builtin::tvm_context_id())) {
       return make_zero(op->dtype);
+    } else if (op->op.same_as(builtin::mem_copy())) {
+      return MakeMemCopy(op);
     } else {
       return StmtExprMutator::VisitExpr_(op);
     }
   }
+
+  PrimExpr MakeMemCopy(const CallNode* op) {
+    PrimExpr dst = op->args[0];
+    PrimExpr src = op->args[1];
+    PrimExpr size = op->args[2];
+
+    std::string fdevapi_prefix = "device_api.";
+    fdevapi_prefix += runtime::DeviceName(device_type_.as<IntImmNode>()->value);
```

Comment:
can just initialize `std::string fdevapi_prefix = "device_api." + runtime::DeviceName(device_type_.as<IntImmNode>()->value);`

Reply:
Did this:
```
std::string fdevapi_prefix = "device_api." + std::string(runtime::DeviceName(device_type_.as<IntImmNode>()->value));
```
Because runtime::DeviceName returns a char*.